recipes/jupyter-book/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "jupyter-book" %}
+{% set version = "0.7.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 92bb13b517568498e5176c62d088ec9430a9059ee6505414ca7663405ceb1686
+
+build:
+  number: 0
+  entry_points:
+    - jb = jupyter_book.commands:main
+    - jupyter-book = jupyter_book.commands:main
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - attrs >=19.3,<20
+    - click
+    - docutils >=0.15
+    - myst-nb >=0.8.0,<0.9
+    - nbconvert
+    - nbformat
+    - python >=3.6
+    - pyyaml
+    - sphinx <3
+    - sphinx-comments
+    - sphinx-copybutton
+    - sphinx-thebe >=0.0.6
+    - sphinx_book_theme >=0.0.34
+    - sphinx_togglebutton
+    - sphinxcontrib-bibtex
+
+test:
+  imports:
+    - jupyter_book
+    - jupyter_book.commands
+    - jupyter_book.directive
+  commands:
+    - jb --help
+    - jupyter-book --help
+
+about:
+  home: "https://executablebooks.org/"
+  license: BSD-3-Clause
+  license_file: LICENSE
+  summary: "Jupyter Book: Create an online book with Jupyter Notebooks"
+  doc_url: https://jupyterbook.org
```

Comment:
it's also nice to have the repo url:

```yaml
dev_url: https://github.com/executablebooks/jupyter-book
```

and/or even the docs (though I don't know if service/tool is doing anything with this yet):

```
doc_source_url: https://github.com/executablebooks/jupyter-book/tree/v{{ version }}/docs
```